savant/app/results.py

```diff
@@ -36,8 +36,8 @@ def compare():
 
     return new_packages
 
-@app.route('/doveps/api/action/create/<json_file>/<name>/<resource>/<action>', methods=['GET'])
```

Comment:
You still need to accept these parameters; see #49

Reply:
When you used Vagrant to install apache2, it ran "apt-get install apache2". So for knowledge for this API request:
* name = apache2
* resource = package
* action = install

Reply:
So in the table, you need multiple rows:
* name "apache2", resource "package", action "install", packages table id "1" (for example)
* name "apache2", resource "package", action "install", packages table id "2" (for example)
* name "apache2", resource "package", action "install", users table id "1" (for example)